Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,65 @@
+
+## Selectors
+
+The `$` command is a short way to call the findElement command in order to fetch a single element on the page
+
+| Name                                     | Protractor                               |WebdriverIO           |
+| -----------------------------------------| -----------------------------------------|----------------------|
+| By class name                            |`element(by.css('.className'))`           |`$('.className')`     |
+| Element containing certain string Text   |`element(by.cssContainingText(tag, text))`|`$('tag=text')`       |
+| Multiple Elements                        |`element.all(by.css('.className'))`       |`$$('.className')`    |
+| By ID                                    |`element(by.css('#idName'))`              |`$('#idName')`        |
+| Chain Selectors                          |`element(by.css('some-css')).element(by.css('tag-within-css'))`  | `$('some-css').$('tag-within-css’)`      |
+| Selecting element one by one in loop     |`element.get(i)`                          |`$$(selector)[i]`     |
+
+There is no proper substitute available for the command `element.getWebElement()` in webdriverIO but we can use `browser.findElement()` as a substitute for this.
+
+## Actions
+
+| Actions                                  | Protractor                               |WebdriverIO            |
+| -----------------------------------------| -----------------------------------------|-----------------------|
+| Click on the element                     |`element.click()`                         |`$(selector).click()`  |
+| Get the text content from a DOM-element  |`element.getText())`                      |`$(selector).getText()`|
+| Navigate to the given destination        |`browser.get(url)`                        |`browser.url(url)`     |
+| Get the url of the currently opened website.|`browser.getCurrentUrl()`              |`browser.getUrl()`     |
+| Get an attribute from a DOM-element based on the attribute name.|`element.getAttribute(attributeName)`|`$(selector).getAttribute(attributeName)`|
+| Wait until a given condition is fulfilled.|`browser.wait(condition, time, optional_msg)`|`browser.waitUntil(condition, { timeout, timeoutMsg, interval })`|
+| Send a sequence of keystrokes to the active element|`element.sendKeys(keys)` |`browser.keys(value`     |
+| Selected DOM-Element is clickable, visible, and exists.|`until.elementToBeClickable(element)` |`$(selector).isClickable()`|
+|`Get the value of a <textarea>, <select> or text <input> found by given selector`|`element.getText()` |`$(selector).getValue()`|
+|`Clear a <textarea> or text <input> element’s value`   |`element.clear()` |   `$(selector).clearValue()`  |
+
+For more details please visit [ webdriverio ](https://webdriver.io/docs/gettingstarted) official documenation page.
+
+## Points to Note While Migration
+
+1. `expect` calls must be prefixed with an `await`.
+
+Example of **incorrect** code for this rule:
```

Comment:
Please indent all the step 1 text to the same level as the first line (`expect` calls must be ...)

Reply:
Done.

Reply:
Not done. You can see that this isn't right by looking at the rendered markdown https://github.com/oppia/oppia-web-developer-docs/blob/6d9f685778d96fbd72e8a0e9e6c8ac09c8f8202e/Guide-to-migrate-e2e-tests.md

Reply:
Now there is no need for indentation at this point as I have removed this.